src/i18n/languages.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
   pl: 'Polish',
   ru: 'Русский',
   ua: 'Українська',
+  'zh-Hant': 'Chinese (Traditional)',
```

Comment:
I'm trying to figure out what would be the best and consistent way to display non-Latin languages in the language selection. Does it make sense to display a language like Chinese, Russian, Greek, Ukrainian, Thai, ... in English? Should we do both? 

What would be your preferred way when you see an language selection?

Reply:
reference: PR #69. I think it's a better way to keep language key in English.